nostr/relay_manager.py

```diff
@@ -1,8 +1,19 @@
+import json
 import threading
+
+from .event import Event
 from .filter import Filters
 from .message_pool import MessagePool
+from .message_type import ClientMessageType
 from .relay import Relay, RelayPolicy
 
+
+
+class RelayException(Exception):
```

Comment:
What do you think about creating an `exception.py` file and putting this class in it? I'm thinking there will be more exceptions created in the future so might as well organize them in one place.

Reply:
I don't know what's the best practice on that, but I've always kept exception classes alongside the area of the code that would `raise` them. This sort of naming convention (`Relay` can raise `RelayException` and both are imported from the `relay` module) just always made sense to me.

Reply:
Okay, I'm fine with keeping it as is.